packages/block-library/src/shortcode/editor.scss

```diff
@@ -1,9 +1,3 @@
-[data-type="core/shortcode"] {
-	&.components-placeholder {
-		min-height: 0;
-	}
-}
```

Comment:
This style is intended to override the Placeholder's minimum height. However, since `min-height` has been removed in [#54216](https://github.com/WordPress/gutenberg/pull/54216/files#diff-e6d39e2bf2050a2e7a7022d39b3ddb498bf87165293fd8759302bab4c2dfb346L6), it is no longer needed.